mu/modes/adafruit.py

```diff
@@ -64,6 +64,10 @@ class AdafruitMode(MicroPythonMode):
         (0x239A, 0x803C),  # future board
         (0x239A, 0x803E),  # future board
         (0x239A, None),    # Any Adafruit Boards
+
+        # Non-Adafruit boards
+        (0x1B4F, 0x8D22),  # SparkFun SAMD21 Mini Breakout
+        (0x1B4F, 0x8D23),  # SparkFun SAMD21 Dev Breakout
```

Comment:
Is this Vendor ID specific to Sparkfun boards?
(i.e. although probably not applicable for this case, some boards use the same type of UART-to-USB bridges, so they could end up with the same VID/PID).

Reply:
Yes, the VID is unique to SparkFun. I had @jimblom from SparkFun pull the PIDs for the boards specifically for CircuitPython.

Reply:
Great, thanks @ShawnHymel.